lib/jsonwp-proxy/proxy.js

```diff
@@ -26,15 +26,15 @@ class JWProxy {
   }
 
   endpointRequiresSessionId (endpoint) {
-    return !_.includes(['/session', '/sessions', '/status'], endpoint);
+    return !_.includes(['/session', '/sessions', '/status', '/homescreen'], endpoint);
```

Comment:
Why is this a sessionless command? I understand that it is in Facebook's [WDA](https://github.com/facebook/WebDriverAgent/blob/2031c1ccfaf7242cf48e6086fe58a9f46ed3c3ed/WebDriverAgentLib/Commands/FBCustomCommands.m#L32), but I'm wondering of the actual use case.